Source/ToolCore/JSBind/JSBTypeScript.cpp

```diff
@@ -57,6 +65,9 @@ String JSBTypeScript::GetScriptType(JSBFunctionType* ftype)
         scriptType = "string[]";
     }
 
+    if (ftype->type_->asHeapPtrType())
+        scriptType = "any";
+
```

Comment:
JSBind should be filtering methods it doesn't have proper typing on, an example or two would help here

I would say "any" is definitely better then a superfluous "number" typing!